sdk/eventhub/Azure.Messaging.EventHubs/tests/Samples/SamplesLiveTests.cs

```diff
@@ -46,11 +59,33 @@ public class SamplesLiveTests
         [TestCaseSource(nameof(SampleTestCases))]
         public async Task SmokeTestASample(IEventHubsSample sample)
         {
-            await using (EventHubScope scope = await EventHubScope.CreateAsync(2))
+            await using (var scope = await EventHubScope.CreateAsync(2))
```

Comment:
*slow clap*

Damn static analysis tool that was run really doesn't appreciate `var` and was overly aggressive about it.